lib/klaviyo_api/resources/profile_exclusion.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'addressable/uri'
+
+module KlaviyoAPI
+  class ProfileExclusion < Base
+    extend KlaviyoAPI::Support::Countable
+
+    self.prefix += 'v1/people/'
+
+    self.element_name = 'exclusion'
+
+    self.collection_parser = KlaviyoAPI::Collections::PaginatedCollection
+
+    class << self
+      def collection_path(prefix_options = {}, query_options = {})
+        super prefix_options, query_options.deep_merge({ api_key: headers['api-key'] })
+      end
+    end
+
+    def create
+      run_callbacks :create do
+        # This endpoint does not accept JSON bodies.
```

Comment:
Maybe toss this comment before the method definition?

Reply:
To me this is more of an implementation concern rather than an api concern, and I usually put those in function bodies. Same thing was done for profile#updates. Should I move them both?